Documentation/ExtensionArchitecture/HowTo/BackendModule/CreateModuleWithExtbase.rst

```diff
@@ -62,8 +62,7 @@ After that you can add titles, menus and buttons using :php:`ModuleTemplate`:
         $this->view->assign('someVar', 'someContent');
         $moduleTemplate = $this->moduleTemplateFactory->create($this->request);
         // Adding title, menus, buttons, etc. using $moduleTemplate ...
```

Comment:
Does this comment still apply? Or should we say "Render the specific template of your action (which can include partials and all the backend assets like menus, buttons and partials)

Reply:
Yes, this comment still applies, since adding buttons, menues etc. must be performed using the ModuleTemplate object.

Reply:
@garvinhicking is your comment resolved?

Reply:
Comment of this patch is OK for me. Maybe I would write "ButtonBar" instead for better differentiation between "a button in my template" and "a button in buttonbar of my backend module". The solution of @garvinhicking reads like "menus" and "buttons" in your fluid template which is not the idea of that original comment IMO.

Reply:
Yes sorry. Lets merge it like this, we can address improvements in followup.